src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -79,15 +97,24 @@ const TeamMembersPageBlock: React.FC<TeamMembersPageBlockProps> = ({ team }) =>
     <PageBlock title={'People'} headerRight={editButtons}>
       <Grid container spacing={1}>
         <Grid item xs={12}>
-          <DetailDisplay label="Head" content={fullNamePipe(team.head)} />
+          <FormLabel>Head</FormLabel>
+          <NERAutocomplete
+            id="head-autocomplete"
+            options={headOptions.map((option) => ({ id: `${option.id}`, label: option.label }))}
```

Comment:
Just make the headOptions match this id rather parse the int out of the option than have to recast everything like this